add.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Add New Products</title>
+</head>
+
+<body background="image/images.jpg">
+    <nav class="topnav">
+        <ul>
+            <li><a href="view_products.html">Products</a></li>
+            <li><a href="sales.html">Sales</a></li>
+            <li><a href="add.html">Add</a></li>
+            <li><a href="index.html">Logout</a></li>
+        </ul>
+    </nav>
+    <main class="boxed">
+        <h1>New Product</h1>
+        <form action="view_products.html">
+            <label for="Product Name">PRODUCT NAME</label><br>
+            <input id="name" name="product_name" type="text" placeholder="Product Name" /><br>
+            <label for="Product Id">PRODUCT ID</label><br>
+            <input id="productid" name="product_id" type="text" placeholder="Product Id" /><br>
+            <label for="Quantity">QUANTITY</label><br>
+            <input id="quantity" name="quantity" type="text" placeholder="Quantity" /><br>
+            <label for="Brand">BRAND</label><br>
+            <input id="brand" name="brand" type="text" placeholder="Brand" /><br>
+            <label for="Price">PRICE</label><br>
+            <input id="price" name="price" type="text" placeholder="Price" /><br><br>
+            <button class="nav" type="submit">Add to cart</button>
```

Comment:
🙅‍♂️ This is a bad way of naming and using classes. You should name classes to match names of what they do. It's better you create another class named button to use for buttons. If you are building a big application what you are doing becomes bad as you might change the `nav` class thinking that you've only changed the `nav` yet you are changing buttons and other elements using the nav class